src/firebaseConfig.ts

```diff
@@ -45,7 +46,7 @@ export type HostingConfig = {
     function?: string;
     run?: {
       serviceId: string;
-      region?: string;
+      region?: "us-central1";
```

Comment:
I am not sure about this, but currently Hosting rewrites can only rewrite to `us-central1` resources.